Src/AutoFixture/ElementsBuilder.cs

```diff
@@ -0,0 +1,57 @@
+﻿using Ploeh.AutoFixture.Kernel;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Ploeh.AutoFixture
+{
+    /// <summary>
+    /// Draws a random element from the given collection
+    /// </summary>
+    /// <typeparam name="T"></typeparam>
+    public sealed class ElementsBuilder<T> : ISpecimenBuilder
+    {
+        private readonly ICollection<T> elements;
+        private readonly RandomNumericSequenceGenerator sequence;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ElementsBuilder{T}"/> class.
+        /// </summary>
+        /// <param name="elements">The collection from which elements should be drawn from.
+        /// It must contain at least two element.</param>
+        public ElementsBuilder(ICollection<T> elements)
```

Comment:
I choose ``ICollection<T>`` because the implementation needs to know the number of elements in the "collection". I considered using ``IEnumerable<T>`` instead, but the Count() extension method because it might trigger side effects.